### PR TITLE
[Coinbase, Common, Core, Injected-Wallets, Keepkey, Keystone, Ledger, React, Trezor, Vue] Release - Master

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
Public release for Packages:

-   Coinbase v2.0.3
-   Common v2.1.0
-   Core v2.2.9
-   Injected-Wallets v2.0.7
-   Keepkey v2.1.0
-   Keystone v2.1.0
-   Ledger v2.1.0
-   React v2.1.7
-   Trezor v2.1.0
-   Vue 2.0.0

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -41,15 +41,15 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@web3-onboard/common": "^2.0.7",
+    "@web3-onboard/common": "^2.1.0-alpha.2",
```
Comment (aaronbarnardsound) [inline packages/core/package.json:44]: Should the alpha be removed here for release?

Comment (Adamj1232) [reply packages/core/package.json]: Good eye - Fixed!

File: packages/keystone/package.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@web3-onboard/common": "^2.0.7",
     "@ethereumjs/tx": "^3.4.0",
     "@ethersproject/providers": "^5.5.0",
-    "@keystonehq/eth-keyring": "^0.11.2-alpha.2"
+    "@keystonehq/eth-keyring": "^0.14.0-alpha.10.3",
```
Comment (aaronbarnardsound) [inline packages/keystone/package.json:23]: Remove alpha from here as well?

Comment (Adamj1232) [reply packages/keystone/package.json:23]: I'm not too sure on this one. @taylorjdawson thoughts?

**Review by aaronbarnardsound — APPROVED** [review]
Looks good!
Just a couple of questions around versioning to double check they are the intended versions.

File: packages/keystone/package.json (hunk 2 above)
Comment (aaronbarnardsound) [reply packages/keystone/package.json:23]: Oh yes actually this is an outside dep. Nevermind!

**Review by mahmud-bn — APPROVED** [review]

**Pull request merged**